app/src/test/java/com/nvapp/navigation/destination/data/PhoneNumberTest.java

```diff
@@ -0,0 +1,44 @@
+package com.nvapp.navigation.destination.data;
+
+import android.net.Uri;
+import android.telephony.PhoneNumberUtils;
+
+import com.navapp.navigation.destination.data.PhoneNumber;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(manifest = Config.NONE)
+public class PhoneNumberTest {
+
+    @Test(expected = IllegalArgumentException.class)
+    public void create_tooShortNumber_throwsException() {
+        new PhoneNumber("12345");
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void create_tooLongNumber_throwsException() {
+        new PhoneNumber("1234567890123456");
+    }
+
+    @Test
+    public void create_normal_returnsSameNumberWithTel() {
+        final String NUMBER = "123456";
+        PhoneNumber phoneNumber = new PhoneNumber(NUMBER);
+
+        assertEquals(Uri.parse("tel:"+NUMBER), phoneNumber.getCallUri());
+    }
+
+    @Test
+    public void create_normalWithLetters_returnsNormalizedNumberWithTel() {
+        final String NUMBER = "123456GAY";
+        PhoneNumber phoneNumber = new PhoneNumber(NUMBER);
+
+        assertEquals(Uri.parse("tel:"+PhoneNumberUtils.normalizeNumber(NUMBER)), phoneNumber.getCallUri());
```

Comment:
You're again perform actions to produce the expected result, instead of specifying what it should be.
The expected data should be something known. Here, it depends on `PhoneNumberUtils`, this defeats the purpose of the test.

Reply:
yeah makes sense